app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -22,12 +22,12 @@ def linkedin
     if @user && @user.persisted?
       sign_in @user, event: :authentication #this will throw if @user is not activated
       set_flash_message(:notice, :success, kind: "LinkedIn") if is_navigational_format?
-      redirect_to employer_home_path
+      redirect_to edit_employer_path(current_user.employer)
```

Comment:
My thought was that with the /employers page gone, there was no obvious link to edit the employer profile, so bouncing them straight to the edit employer page would be good. When logged in, the button on the commitments page takes you to an anchor on the same edit employer page, so maybe as a compromise, change "Update your commitment" to "Update your employer profile and commitment", and then logging in can redirect to /commitments.?